apps/frontend/src/components/Composite/booking/confirmation/ConfirmedSessionCard.tsx

```diff
@@ -16,15 +16,17 @@ export const ConfirmedSessionCard = ({
   endTime,
 }: ConfirmedSessionCardProps) => {
   return (
-    <Card className="bg-white/10 text-primary-foreground">
-      <p className="font-medium text-lg">{weekDay}</p>
-      <div className="text-primary-foreground/70 text-sm">
-        <p className="mb-2">
-          {startTime} - {endTime}
-        </p>
-        <p>{locationName}</p>
-        <p>{address}</p>
-      </div>
+    <Card variant="solid">
+      <CardHeader fontSize="large" fontWeight="medium">
+        {weekDay}
+      </CardHeader>
+      <CardBody color="whiteAlpha.700" fontSize="small">
+        {startTime} - {endTime}
+        <br />
+        {locationName}
+        <br />
+        {address}
+      </CardBody>
```

Comment:
I think it's better to use `lg` instead of `large`, and `sm` instead of `small`.
We can use `Heading` in the card header.

```suggestion
      <CardHeader>
        <Heading.h3>{weekDay}</Heading.h3>
      </CardHeader>
      <CardBody gap="sm">
        <Text>{startTime} - {endTime}</Text>
        <Text>{locationName}</Text>
        <Text>{address}</Text>
      </CardBody>
```